packages/trip-form/src/settings-selector-panel.story.js

```diff
@@ -93,3 +95,51 @@ export const settingsSelectorPanelUndefinedParams = () => (
     />
   </PanelWrapper>
 );
+
+const Button = styled.button`
+  display: block;
+  margin-bottom: 10px;
+`;
+
+const FooterContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 20px 0px;
+  > button {
+    width: 66%;
+    cursor: pointer;
+  }
+  > .line-break {
+    width: 100%;
+  }
+`;
+
+export const tripOptions = () => (
```

Comment:
IMO, there should be stories that show the basic states of the subcomponents (e.g. checkboxes, the top band, rental company selector, transit mode selector).

Reply:
I don't know if these subcomponents will ever exist on their own? @landonreed ?